python/ray/util/sgd/torch/distributed_torch_runner.py

```diff
@@ -42,6 +42,7 @@ def setup(self, url, world_rank, world_size):
 
     def _setup_distributed_pytorch(self, url, world_rank, world_size):
         self.world_rank = world_rank
+        self.config["world_rank"] = world_rank
```

Comment:
We should really come up with a better API for passing system properties around at some point :P Just noting this for the future.

Reply:
I think I'll actually be able to rework the code and get rid of this, but I agree it's a bit clunky in general.